CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+* Backend planners now use multi-inherence instead of `__call__` to include the backend functions. This allows for better generated documentation.
```

Comment:
```suggestion
* Backend planners use multi-inherence instead of `__call__` to include the backend functions. This allows for better generated documentation.
```